scripts/common/yaml.sh

```diff
@@ -125,7 +125,7 @@ function apply_installer_crd() {
         cat > $ORIGINAL_INSTALLER_SPEC <<EOL
 ${INSTALLER_YAML}
 EOL
-        kubectl apply -f "$ORIGINAL_INSTALLER_SPEC"
+        try_1m kubectl apply -f "$ORIGINAL_INSTALLER_SPEC"
```

Comment:
The line just apply the manifest.
The try will be re-doing the same command each 2 seconds : https://github.com/replicatedhq/kURL/blob/c09c7fb818be8cfab5d5d51b0a1e2bd26c4d9a59/scripts/common/common.sh#L440-L458

So, why would need to be required to re-duo kubectl apply many times? 
What would be solved with that?

+1 to the @emosbaugh comments. That would be great if you could share the problem, and all logs of the issue that you are facing so we can better check it out and see how we can help.